src/components/Cards/Results.vue

```diff
@@ -0,0 +1,330 @@
+              <td class="font-bold text-lg">Bi-Monthly Repayment</td>
+              <td class="font-bold text-lg">Total Repayment</td>
+              <td class="font-bold text-lg">Total Product Price</td>
+            </thead>
+            <tbody
+              v-for="(downpayments, index) in computedGetCalc"
+              :key="index"
+            >
+              <tr
+                class="
+                  my-2
+                  px-3
+                  py-2
+                  flex
+                  justify-evenly
+                  items-center
+                  space-x-20
+                "
+                v-if="downpayments.bizId == b_type.id"
+                :class="
+                  downpayments.percent == 20
+                    ? 'bg-blue-100'
+                    : downpayments.percent == 40
+                    ? 'bg-red-100'
+                    : downpayments.percent == 60
+                    ? 'bg-green-100'
+                    : 'bg-yellow-100'
+                "
+              >
+                <td class="font-bold text-lg -mr-6 -ml-10">
+                  {{ downpayments.percent }}%
+                </td>
+                <td class="font-medium text-base">
+                  {{
+                    $formatCurrency(
+                      Math.floor(downpayments.actualDownpayment)
+                    ) || ""
+                  }}
+                </td>
+                <td class="font-medium text-base">
+                  {{
+                    $formatCurrency(
+                      Math.floor(downpayments.biMonthlyRepayment)
+                    ) || ""
+                  }}
+                </td>
+                <td class="font-medium text-base">
+                  {{
+                    $formatCurrency(Math.floor(downpayments.actualRepayment)) ||
+                    ""
+                  }}
+                </td>
+                <td class="font-medium text-base">
+                  {{ $formatCurrency(Math.floor(downpayments.total)) || "" }}
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </div>
+ </div>           
+</template>
+
+<script>
+export default {
+    data(){
+        return{
+            id:this.$route.params.name
+        }
+    },
+    computed:{
+         toggleColor() {
+      let color = "";
+      let selectDownpayment = !this.selectedDownpayment
+        ? null
+        : this.selectedDownpayment.percent;
+      switch (selectDownpayment) {
+        case 20:
+          color = "bg-blue-300";
+          break;
+        case 40:
+          color = "bg-red-300 ";
+          break;
+        case 60:
+          color = "bg-green-300 ";
+          break;
+        case 80:
+          color = "bg-yellow-300 ";
+          break;
+        default:
+          color = "bg-gray-400 ";
+      }
+      return color;
+    },
+    toggletextColor() {
+      let color = "";
+      let selectDownpayment = !this.selectedDownpayment
```

Comment:
Change line 275 to `let selectDownpayment = this.selectedDownpayment?.percent;`

Reply:
@cheedinmar what about this?

Reply:
@cheedinmar did you not see this?

Reply:
@teebarg  done